src/infrastructure/database/crud.py

```diff
@@ -79,6 +79,7 @@ async def _get(self, key: str, value: Any) -> ConcreteSchema | None:
             getattr(self.schema_class, key) == value
         )
         results = await self._execute(query=query)
+        await self.session.commit()
```

Comment:
why again added commit?

Reply:
For some reason flush is not working for me :)
You can pull this branch and try by yourself

Reply:
I removed it from BaseCRUD _get

Reply:
you can close this issue, I have fixed it